utils/tempered.c

```diff
@@ -30,6 +31,9 @@ void show_help()
 "Known options:\n"
 "    -h\n"
 "    --help                 Show this help text\n"
+"    -t\n"
+"    --type                 Returns only the numeric value of the sensor.\n"
+"                           Known sensors: temperature, humidity, dew point\n"
```

Comment:
The dew point is calculated from temperature and humidity values, therefore the word "sensor" should probably not be used here. Furthermore, dew point needs to be passed as argument inside quotes. It would make sense to mention this here. Suggestion:
`Known sensors: temperature, humidity, dew point\n"`
->
`Known types: temperature, humidity, \"dew point\"\n"`